### PR TITLE
new arm designator api

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Comment by LoyVanBeek** [comment]
Would be nice if these are all the changes we need :-)

**Comment by MatthijsBurgh** [comment]
It wasn't

File: action_server/src/action_server/actions/place.py [hunk 1]
```diff
@@ -128,20 +124,18 @@ def _configure(self, robot, config):
         return
 
     def _start(self):
+        print("place start")
```
Comment (LarsJanssenTUe) [inline action_server/src/action_server/actions/place.py]: Get rid of this print!

**Comment by MatthijsBurgh** [comment]
This doesn't need to be tested on the robot. This is pure logic.

**Pull request merged**